consensus/scheme/rolldpos/rolldpos.go

```diff
@@ -573,12 +573,13 @@ func (b *Builder) Build() (*RollDPoS, error) {
 		b.clock = clock.New()
 	}
 	ctx := rollDPoSCtx{
-		cfg:     b.cfg,
-		addr:    b.addr,
-		chain:   b.chain,
-		actPool: b.actPool,
-		p2p:     b.p2p,
-		clock:   b.clock,
+		cfg:                    b.cfg,
```

Comment:
@FishMeat are you using golang 1.11.x? 1.11 seems to use a different format scheme compared to 1.10. We're still using 1.10 for development. If we mixed different versions of golang SDK. We will have problem of formatting the following code back and forth.

Reply:
I am using  1.10.3 version.

Reply:
thx for confirmation